src/miner.h

```diff
@@ -7,13 +7,9 @@
 #define NOVACOIN_MINER_H
 
```

Comment:
What is your opinion on all of the include guards? We have names from Bitcoin, Novacoin, etc. Should we change all of these to Gridcoin?

Reply:
I changed any remaining include guards to `#pragma once` in the Gridcoin headers to follow what we've done so far. I think we should leave the Bitcoin include guards alone&mdash;no real benefit to changing those and it can add a minor hassle for upstream merges if we do.

For this review item in particular, I didn't do anything with the miner this round, but the header will change in the future.